src/api/app/assets/stylesheets/webui/requests.scss

```diff
@@ -13,6 +13,10 @@
   background-color: $cyan;
 }
 
+.badge-warning {
```

Comment:
This will overwrite the default `badge-warning` class from Bootstrap, see https://getbootstrap.com/docs/4.6/components/badge/#contextual-variations. We already use that class in other views, so we should avoid this.

Instead, we can combine `badge` with `alert-warning` and achieve the same.